<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="DpyWidget" name="dpy" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnF1">
        <property name="text">
         <string>Start
Evolution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnF2">
        <property name="text">
         <string>Focal
Length -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnF3">
        <property name="text">
         <string>Focal
Length +</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnF5">
        <property name="text">
         <string>First
Generation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnF6">
        <property name="text">
         <string>Generation
-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnF7">
        <property name="text">
         <string>Generation
+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnF8">
        <property name="text">
         <string>Play
Generations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnF10">
        <property name="text">
         <string>Quit
Program</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DpyWidget</class>
   <extends>QWidget</extends>
   <header>dpywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
